_inc/client/at-a-glance/scan.jsx

```diff
@@ -106,51 +108,75 @@ class DashScan extends Component {
 	};
 
 	getVPContent() {
-		const { sitePlan, planClass, fetchingSiteData } = this.props;
-		const hasSitePlan = false !== sitePlan;
-		const vpData = this.props.vaultPressData;
-		const scanEnabled = get( vpData, [ 'data', 'features', 'security' ], false );
+		const { vaultPressData } = this.props;
 
-		if ( this.props.getOptionValue( 'vaultpress' ) ) {
-			if ( 'N/A' === vpData ) {
-				return renderCard( {
-					status: '',
-					content: __( 'Loading…', 'jetpack' ),
-				} );
+		// The VaultPress plugin is active but not registered, or we can't connect
+		if ( vaultPressData?.code === 'not_registered' ) {
+			return renderCard( {
+				className: 'jp-dash-item__is-inactive',
+				status: 'not-registered',
+				content: jetpackCreateInterpolateElement(
+					__(
+						"VaultPress is having trouble connecting. Make sure you've <keyLink>entered your registration key</keyLink>, or <supportLink>contact support</supportLink> if you still need help.",
+						'jetpack'
+					),
+					{
+						keyLink: <a href="admin.php?page=vaultpress" />,
```

Comment:
Maybe we could use `siteAdminUrl` from the props here, to ensure we always build the right URL?